app/components/post_list/post_list.js

```diff
@@ -48,6 +49,7 @@ export default class PostList extends PureComponent {
         highlightPostId: PropTypes.string,
         initialIndex: PropTypes.number,
         isSearchResult: PropTypes.bool,
+        lastPostIndex: PropTypes.number.isRequired,
```

Comment:
@saturninoabril we forgot about the Thread view and permalink, so those screens are showing a warning where `lastPostIndex` is **undefined**, can we make sure we fix that?

Reply:
PR submitted - https://github.com/mattermost/mattermost-mobile/pull/2780